packages/nimble-components/karma.conf.ci.js

```diff
@@ -1,5 +1,5 @@
 /**
- * A karma config file that adds verbose reporting to the base configuration
+ * A karma config file that sets up reporting for CI builds
```

Comment:
I'd vote for "quiet", but mainly I want to avoid just having the bucket "ci" because that doesn't really tell me anything about how it runs different

Reply:
or maybe "headless"

Reply:
I hadn't gone with headless because we weren't using it for all of our headless commands (`test-chrome`, `test-chrome:watch`, and other browser equivalents). And I was initially reluctant to do so because I thought those commands were run by devs and needed better live logging to the terminal.

But it looks like their live logging to the terminal is also quiet (only prints failing tests, no live feedback about how many tests have passed). So we could consolidate those commands:

`karma.conf.headless.js` called by `test-chrome` and `test-chrome:watch`. Delete `test-chrome:verbose` and update `test-concurrent` to call `test-chrome` instead.

The only behavior change would be that skipped tests would now be printed to the console when you call `test-chrome`, which I think I'm fine with.